core/src/main/java/org/ocpsoft/prettytime/format/SimpleTimeFormat.java

```diff
@@ -118,13 +118,17 @@ protected long getQuantity(Duration duration, boolean round)
    protected String getGramaticallyCorrectName(final Duration d, boolean round)
    {
       String result = getSingularName(d);
-      if ((Math.abs(getQuantity(d, round)) == 0) || (Math.abs(getQuantity(d, round)) > 1))
+      if (isPlural(d, round))
       {
          result = getPluralName(d);
       }
       return result;
    }
 
+   protected boolean isPlural(final Duration d, boolean round) {
```

Comment:
Just a heads up:
I extracted this to a separate method here because I need to do the exact same check inside my custom `DeTimeFormat` and like to prevent duplicate code. I think it might generally come in handy to have this method available when implementing a custom `TimeFormat`.

Reply:
I agree. This is fine. Consumers of individual resource bundles may need this information. They may also wish to override the functionality, which would require some ... fun... but this would at least make things a bit more discoverable. I don't see any problems with exposing this here.